plugins/org.eclipse.fordiac.ide.model/model/lib.ecore

```diff
@@ -1973,7 +1973,34 @@
     </eOperations>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="ServiceInterfaceFBType" eSuperTypes="#//FBType"/>
-  <eClassifiers xsi:type="ecore:EClass" name="SimpleFBType" eSuperTypes="#//BaseFBType"/>
+  <eClassifiers xsi:type="ecore:EClass" name="SimpleECAction">
+    <eStructuralFeatures xsi:type="ecore:EReference" name="algorithm" eType="#//Algorithm"/>
+    <eStructuralFeatures xsi:type="ecore:EReference" name="output" eType="#//Event"/>
+  </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="SimpleECState" eSuperTypes="#//INamedElement #//PositionableElement">
+    <eStructuralFeatures xsi:type="ecore:EReference" name="simpleECAction" upperBound="-1"
```

Comment:
Please use the plural form for one-to-many references:
```suggestion
    <eStructuralFeatures xsi:type="ecore:EReference" name="simpleECActions" upperBound="-1"
```

Reply:
changed